src/Button/types.ts

```diff
@@ -48,6 +48,8 @@ export type ButtonProps = {
 
 export type IconButtonProps = ButtonA11yProps & {
   icon: React.FunctionComponent<IconProps>
+  'aria-label': string
+  disableTooltip?: boolean
```

Comment:
Should we document this prop?

Reply:
Not sure, I didn't want to include it in the docs because we don't want to encourage folks to use it, but we would need it for internal components like NavList and TreeView. 

more in the issue thread: [primer/react#2008](https://github.com/primer/react/issues/2008#:~:text=Avoid%20multiple%20tooltips%20automatically%2C%20don%27t%20add%20hideTooltip%20to%20public%20API%20just%20yet)